jobs/JGDAS_ATMOS_VERFOZN

```diff
@@ -4,36 +4,12 @@
 # Set up environment for GDAS Ozone Monitor job
 #############################################################
 source "${HOMEgfs}/ush/preamble.sh"
-
-###########################################################
-# make temp directories
-###########################################################
-export DATA=${DATA:-${DATAROOT}/${jobid}}
-mkdir -p ${DATA}
-cd ${DATA}
-
-
-##############################################
-# Run setpdy and initialize PDY variables
-##############################################
-export cycle="t${cyc}z"
-setpdy.sh
-. ./PDY
-
-
-####################################
-# Determine Job Output Name on System
-####################################
-export pid=$$
-export pgmout="OUTPUT.${pid}"
-export pgmerr=errfile
+source "${HOMEgfs}/ush/jjob_header.sh" base vrfy
```

Comment:
This job does not source any configs.  We are now forcing it to source these files that may or may not be applicable for this job.
Likewise, it is going to source vrfy section in $machine.env

Reply:
It did, at least in development, but it was being done in the rocoto pre-job (`jobs/rocoto/vrfy.sh`) (and still is, for now). Added it here to be consistent with other j-jobs, since the job should be broken up so that only one j-job is called per rocoto task.